src/views/calculator/index.vue

```diff
@@ -73,6 +74,14 @@ export default {
            registerBg2
            
        }
-   } 
+   } ,
+   methods:{
+    toggleModal(){
+      this.modal =!this.modal;
+    },
+     onModalClose (){
+      modalState.value = false
```

Comment:
which variable is `modalState `?

Reply:
A mistake, I will get it updated on the next PR